lib/workers/pr/changelog/release-notes.ts

```diff
@@ -157,11 +187,20 @@ export async function getReleaseNotesMd(
   }
   let changelogFile: string;
   let changelogMd = '';
+  let apiTree: string;
+  let apiFiles: string;
+  let filesRes;
```

Comment:
```suggestion
  let filesRes: { name: string }[];
```

Reply:
Not sure about "name", probably should be "body"?

Reply:
`let filesRes: { body: { name: string}[] };`
might be

Reply:
Fixed.